site/public/css/polls.css

```diff
@@ -45,6 +45,10 @@
     font-size: 2em;
 }
 
+.table td {
+    vertical-align: middle !important; 
+}
+
```

Comment:
It would be better to use a class instead of blindly applying this styling to every table on the page.  Although it's not necessarily an issue now, if could be problematic if someone added another table later and wanted to style it differently.